arm64 support

adamhurm opened the pull request:
I was able to add arm64 support to the build process, but it only supports local builds since the version uploaded to dockerhub will be dependent on the build platform architecture.
Docker does support [named build stages](https://docs.docker.com/develop/develop-images/multistage-build/), but the build steps need to be sequential and don't really work like an if-then. This kind of behavior might be possible in [BuildKit](https://docs.docker.com/develop/develop-images/build_enhancements/), but I have never used it before and did not want to modify the build process. The whole build is just wrapped in a bash if-then.

Notes:
- package version numbers were updated if the maintainer added arm64 releases, but I have not fully tested the support of each tool.
- not all package versions were updated.
- amicontained arm64 support is hacky and needs improvement.

madhuakula commented:
Hi @adamhurm,

You are such a fantastic person. Thank you so much for all the efforts to make the community great and for contributing to these 🙏 

I have done some fixes after testing. Could you please confirm if they all look great for ARM as well?

Thank you so much once again and apologies for the delay in response.

adamhurm commented:
Awesome, thanks for the update Madhu! I'm away from home at the moment, but I should be able to test when I return this week.

madhuakula commented:
Take your time to enjoy. No hurry :) 

Thanks a million once again for a wonderful work 💯

adamhurm commented:
Confirmed that everything looks good on arm64 now.
I updated the kubeaudit URL since it just changed during this last release. I also updated my ported arm64 release versions of amicontained and cfssl to match the amd64 release versions.
